src/main/java/com/oltpbenchmark/ThreadBench.java

```diff
@@ -420,9 +420,9 @@ private void calculateNext() {
             while (sample != null && sample.getStartNanosecond() < endNanoseconds) {
 
                 // Check if a TX Type filter is set, in the default case,
```

Comment:
```suggestion
                // Check if a TxType filter is set, in the default case,
```

for consistency with below